File: pipelines/Application1/code/src/main/scala/io/prophecy/pipelines/application1/graph/baseDS1.scala
package io.prophecy.pipelines.application1.graph

import io.prophecy.libs._
import io.prophecy.pipelines.application1.config.ConfigStore._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object baseDS1 {

  def apply(spark: SparkSession): DataFrame =
    spark.read
      .format("csv")
      .option("header",                            true)
      .option("sep",                               ",")
      .schema(StructType(Array(StructField("eded", StringType, true))))
      .load("dede")

}

File: baseProjectPY/pipelines/basePipeline1/code/basepipeline1/graph/baseDS1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from basepipeline1.config.ConfigStore import *
from basepipeline1.udfs.UDFs import *

def baseDS1(spark: SparkSession) -> DataFrame:
    return spark.read\
        .schema(
          StructType([
            StructField("customer_id", StringType(), True), StructField("first_name", StringType(), True), StructField("last_name", StringType(), True), StructField("phone", StringType(), True), StructField("email", StringType(), True), StructField("country_code", StringType(), True), StructField("account_open_date", StringType(), True), StructField("account_flags", StringType(), True)
        ])
        )\
        .option("header", True)\
        .option("sep", ",")\
        .csv("dbfs:/Prophecy/123gkc@gmail.com/CustomersDatasetInput.csv")
